mos-platform/geos-cbm/crt0.c

```diff
@@ -0,0 +1,102 @@
+/***********************************************************************
+ * crt0.c
+ *
+ * This file contains the implemantation of the "C" definition of
+ * init and fini code and the directory entry and info sector for
+ * use in GEOS applications  with llvm-mos C compiler.
+ *
+ ***********************************************************************/
+
+#include "geos.h"
+
+asm(".set __zp_reg_size, 0xfb - 0x90 \n"
```

Comment:
It's easier to read large blocks of asm like this if in their own `.s` files, since it avoids all the C syntactic cruft.

Reply:
Used R"ASM(

That should do it for now